### ISSUE TITLE
#god

### ISSUE THREAD

**Issue by Yunes69790** [j/j.github.com#11]
God is a place in your hear it dont matter how hard you love or dont its always in your heart and you  will feel it when you pray to your god never forgot the prayers.

**Comment by monarosej** [comment]
Yes, God hears your prayers. An ever-present, all-knowing God will hear every word from your mouth, thought in your mind and whisper of your heart. But there are some things that might affect whether He answers. Now, let us get one thing straight before we go on… God loves you.
